src/java/org/broadinstitute/dropseqrna/utils/ConvertTagToReadGroup.java

```diff
@@ -137,9 +137,18 @@ protected int doWork() {
 	 */
 	private SAMReadGroupRecord getReadGroupTemplate(final SAMFileHeader inHeader) {
 		List<SAMReadGroupRecord> recs = inHeader.getReadGroups();
-		String library = recs.get(0).getLibrary();
-		String platform = recs.get(0).getPlatform();
-		String platformUmit = recs.get(0).getPlatformUnit();
+		String library = null;
```

Comment:
@jamesnemesh , 

Can you confirm that my changes to this method look reasonable?

Thanks, Alec

Reply:
@alecw Looks good to me!